README.md

```diff
@@ -10,10 +10,10 @@ Coupled structural solvers written with the C++ finite element library deal.II:
 Applied coupling functionalities have been separated and can be found in the `include/adapter` directory.
 
 ## Start here
-Our [wiki](https://www.precice.org/adapter-dealii-overview.html) will help you start. If you are missing something, [let us know](https://www.precice.org/resources/#contact).
+Our [wiki](https://precice.org/adapter-dealii-overview.html) will help you start. If you are missing something, [let us know](https://recice.org/resources/#contact).
```

Comment:
@DavidSCN we should avoid the "wiki" name here. And we lost the "p" in `https://recice.org`